build_tools/travis/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+echo 'List files from cached directories'
+echo 'pip:'
+ls $HOME/.cache/pip
+
+# Deactivate the travis-provided virtual environment and setup a
+# conda-based environment instead
+deactivate
+
+# Use the miniconda installer for setup of conda itself
+pushd .
+cd
+mkdir -p download
+cd download
+if [[ ! -d /home/travis/miniconda3 ]]
```

Comment:
Given that this is now just a python project, do we even need to worry about this?  Should travis already know how to give us the right python environment for python 3.5?